src/java.base/unix/native/libjli/java_md_common.c

```diff
@@ -82,6 +82,8 @@ GetApplicationHome(char *buf, jint bufsize)
 {
     const char *execname = GetExecName();
     if (execname != NULL) {
+        JLI_TraceLauncher("Launcher executable path is %s\n", execname);
```

Comment:
> This trace seems a bit unsymmetric to its Windows counterpart. Maybe it should be left out here, too, since there is tracing in GetJREPath.

I removed the JLI trace output for the launcher exe path.